pkg/lib/bundle/interfaces.go

```diff
@@ -0,0 +1,21 @@
+package bundle
+
+import (
+	"github.com/operator-framework/operator-registry/pkg/containertools"
+
+	"github.com/sirupsen/logrus"
+)
+
+// BundleImageValidator provides a toolset for pulling and then validating bundle container images
+type BundleImageValidator interface {
+	PullBundleImage(string, string) error
+	ValidateBundle(string) error
+}
+
+// NewImageValidator is a constructor that returns an ImageValidator
+func NewImageValidator(containerTool string, logger *logrus.Entry) BundleImageValidator {
+	return ImageValidator{
```

Comment:
Since this function returns the `BundleImageValidator` interface, does `ImageValidator` need to be exported? If it does need to be exported, would it make sense to return the concrete type and do a separate compile time interface assignment/check. E.g.
```go
var _ BundleImageValidator = &ImageValidator{} 
```